src/screens/TripSettings/TripSettings.tsx

```diff
@@ -14,13 +35,326 @@ type TripItineraryScreenNavigationProp = CompositeNavigationProp<
+                                placeholder={placeholder.tripName}
+                            />
+                            <Input
+                                label={t("description")}
+                                value={values.description ? values.description : ""}
+                                onChangeText={handleChange("description")}
+                                disabled={settingsMode == Mode.VIEW}
+                                onBlur={handleBlur("description")}
+                                multiline={true}
+                                numberOfLines={4}
+                                style={styles.textArea}
+                                errorMessage={
+                                    errors.description && touched.description
+                                        ? errors.description
+                                        : undefined
+                                }
+                                errorStyle={styles.errorMessage}
+                                placeholder={placeholder.description}
+                            />
+                            {errorUpdate && (
+                                <Text style={styles.errorText}>{errorUpdate.message}</Text>
+                            )}
+                            {settingsMode === Mode.EDIT && (
+                                <Button
+                                    containerStyle={styles.buttonContainer}
+                                    buttonStyle={styles.submitButton}
+                                    title={t("screens.tripSettings.submitUpdate")}
+                                    titleStyle={{
+                                        color: "black",
+                                        fontSize: 25,
+                                    }}
+                                    onPress={() => handleSubmit()}
+                                    loading={loadingUpdate}
+                                />
+                            )}
+                        </>
+                    )}
+                </Formik>
+                <Text h3 style={{ marginBottom: 15 }}>
+                    {t("screens.tripSettings.members")}
+                </Text>
+                {data.trip.members.map((member, i) => (
+                    <ListItem key={i} bottomDivider>
+                        <Avatar
+                            rounded
+                            containerStyle={{
+                                backgroundColor: stc(member.user.id),
+                                marginRight: 5,
+                            }}
+                            size="medium"
+                            title={member.user.displayName.charAt(0)?.toUpperCase() ?? "?"}
+                        />
+
+                        <ListItem.Content>
+                            <ListItem.Title>{member.user.displayName}</ListItem.Title>
+                            {member.role === "ADMIN" && (
```

Comment:
```suggestion
                            {member.role === TripUserRole.Admin && (
```
Attention - you need to add the import as well, so accepting it here is not enough.